docker-compose_example/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+services:    
+  rflood:
+    image: binhex/arch-rtorrentvpn
+    container_name: rflood
+    hostname: rflood
+    ports:
+      - "9080:9080"
+      - "9443:9443"
+      - "8118:8118"
+      - "3000:3000"
+      - "9091:9091"
+      - "8888:8888"
+    cap_add:
+      - NET_ADMIN
+    devices:
+      - /dev/net/tun
+    environment:
+      - VPN_ENABLED
+      - VPN_USER
+      - VPN_PASS
+      - VPN_PROV
+      - STRICT_PORT_FORWARD
+      - ENABLE_PRIVOXY
+      - ENABLE_FLOOD
```

Comment:
Where does this ENABLE_FLOOD come from? It's not in the main documentation. Curious because I'd like rflood :)

Reply:
I used to support flood but it was too buggy and lacked a lot of features, so I removed it

Reply:
Ah got it. I was looking for a mobile friendly alternative and rflood seemed to be it. Thanks